packages/crdt/src/crdt/merkle.ts

```diff
@@ -156,7 +171,9 @@ export function debug(trie: TrieNode, k = '', indent = 0): string {
     str +
     getKeys(trie)
       .map(key => {
-        return debug(trie[key], key, indent + 2);
+        const node = trie[key];
+        if (!node) return '';
+        return debug(node, key, indent + 2);
```

Comment:
Another small fix for the typechecker because `node` can be undefined in this context. But I'm not throwing an exception here because it's just a `debug` method.